test/Confluent.Kafka.IntegrationTests/Tests/AdminClient_DeleteConsumerGroupOffsets.cs

```diff
@@ -0,0 +1,101 @@
+            {
+                var groupId = topic1.Name;
+                var offsetToCommit = 10;
+
+                consumer.Assign(new List<TopicPartition>(){new TopicPartition(topic1.Name, 0)});
+                consumer.Commit(new List<TopicPartitionOffset>() {new TopicPartitionOffset(topic1.Name, 0, offsetToCommit)}); //commit some offset for consumer
+
+                var committedOffsets = consumer.Committed(TimeSpan.FromSeconds(10));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(offsetToCommit, committedOffsets[0].Offset);
+
+                List<TopicPartitionOffset> topicPartitionToReset = new List<TopicPartitionOffset>() {new TopicPartitionOffset(topic1.Name, 0, -1)}; //offset value is ignored
+
+                var res = adminClient.DeleteConsumerGroupOffsetsAsync(groupId, topicPartitionToReset).Result;
+                Assert.Equal(1, res.Count);
+                Assert.Equal(groupId, res[0].Group);
+                Assert.Equal(1, res[0].Partitions.Count);
+                Assert.Equal(0, res[0].Partitions[0].Partition.Value);
+
+                committedOffsets = consumer.Committed(TimeSpan.FromSeconds(1));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(-1001, committedOffsets[0].Offset); //offset is unset
+
+                // Consumer is actively subscribed to the topic
+                consumer.Subscribe(new List<String>(){topic1.Name});
+                consumer.Commit(new List<TopicPartitionOffset>() {new TopicPartitionOffset(topic1.Name, 0, offsetToCommit)}); //commit some offset for consumer
+
+                committedOffsets = consumer.Committed(TimeSpan.FromSeconds(10));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(offsetToCommit, committedOffsets[0].Offset);
+
+                topicPartitionToReset = new List<TopicPartitionOffset>() {new TopicPartitionOffset(topic1.Name, 0, -1)}; //offset value is ignored
+                Assert.Equal(1, res.Count);
+                Assert.Equal(groupId, res[0].Group);
+                Assert.Equal(1, res[0].Partitions.Count);
+                Assert.Equal(0, res[0].Partitions[0].Partition.Value);
+
+                committedOffsets = consumer.Committed(TimeSpan.FromSeconds(1));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(10, committedOffsets[0].Offset); //offset is unchanged as the consumer is actively subscribed to the topic
+
+                consumer.Unsubscribe();
+
+                // Deleting offset from the topic which has no set offset
+                consumer.Assign(new List<TopicPartition>(){new TopicPartition(topic2.Name, 0)});
+
+                var watermarkOffsets = consumer.QueryWatermarkOffsets(new TopicPartition(topic2.Name, 0),TimeSpan.FromSeconds(10));
+                Assert.Equal(0, watermarkOffsets.Low);
```

Comment:
watermark offsets have nothing to do with committed offsets. these are the first and last offsets in the log.

Reply:
ACK, update to use committed offsets